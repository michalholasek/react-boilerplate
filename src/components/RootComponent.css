.RootComponent {
  font-family: Helvetica, sans-serif;
}
