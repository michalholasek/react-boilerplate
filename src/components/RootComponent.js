import React from 'react';

import './RootComponent.css';

export default function RootComponent() {
  return <div className="RootComponent">Hello World!</div>;
}
