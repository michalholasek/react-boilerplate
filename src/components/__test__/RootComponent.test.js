import React from 'react';
import renderer from 'react-test-renderer';

import RootComponent from '../RootComponent';

describe('RootComponent', () => {

  it('should render correctly', () => {
    const tree = renderer.create(<RootComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should have one child with correct value', () => {
    const tree = renderer.create(<RootComponent />).toJSON();
    
    expect(tree.children.length).toEqual(1);
    expect(tree.children[0]).toEqual('Hello World!');
  });

});
