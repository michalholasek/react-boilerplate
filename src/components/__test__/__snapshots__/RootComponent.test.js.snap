exports[`RootComponent should render correctly 1`] = `
<div
  className="RootComponent">
  Hello World!
</div>
`;
